tools/cargo-zerocopy/src/main.rs

```diff
@@ -197,12 +197,21 @@ fn install_toolchain_or_exit(versions: &Versions, name: &str) -> Result<(), Erro
 
 fn get_rustflags(name: &str) -> &'static str {
     if name == "nightly" {
-        "--cfg __INTERNAL_USE_ONLY_NIGHTLY_FEATURES_IN_TESTS "
+        "--cfg __ZEROCOPY_INTERNAL_USE_ONLY_NIGHTLY_FEATURES_IN_TESTS "
     } else {
         ""
     }
 }
 
+fn get_toolchain_rustflags(name: &str) -> &'static str {
+    match name {
+        "msrv" => "--cfg __ZEROCOPY_TOOLCHAIN=\"msrv\" ",
+        "stable" => "--cfg __ZEROCOPY_TOOLCHAIN=\"stable\" ",
+        "nightly" => "--cfg __ZEROCOPY_TOOLCHAIN=\"nightly\" ",
+        _ => "",
+    }
+}
```

Comment:
Maybe just unconditionally forward it? We also use [these](https://github.com/google/zerocopy/blob/main/Cargo.toml#L29) toolchain descriptors, which are liable to change over time.

```suggestion
fn get_toolchain_rustflags(name: &str) -> String {
    format!("--cfg __ZEROCOPY_TOOLCHAIN=\"{}\", name)
}
```